assets/src/styles/blocks.scss

```diff
@@ -27,6 +27,7 @@
 @import "blocks/core-overrides/BlockSpacing";
 @import "blocks/core-overrides/Separator";
 @import "blocks/core-overrides/Table";
+@import "blocks/core-overrides/MediaAndText";
```

Comment:
We should probably use the block name for directories, [like Gutenberg does](https://github.com/WordPress/gutenberg/tree/8536f6f1aa1df9daa98193964aa8e62cc196cb7a/packages/block-library/src/media-text).

Reply:
In our case it's not a directory, it's a file, not sure if that changes anything for your comment 😬 Do you mean a structure like `blocks/core-overrides/media-text/style.scss`, so having a folder per core block?

Reply:
No same folder structure, just using the block actual name for the file name `_media-text.scss` instead of creating our own identifier for it.

Reply:
Sure, updated!